onnx/onnxifi_ext.h

```diff
@@ -7,6 +7,11 @@
 extern "C" {
 #endif
 
+static const int extension_function_number = 2;
+static const char extension_function_list[] =
```

Comment:
Why do we need to put these things here? The factor that we include "onnxifi_ext.h" also means we have access to these names, right? Such a table should be kept at the backend side IMO.

Reply:
I also took some consideration when design it like this. My reason of putting these define here is we would like to make this two consts represent the super set of functions we support. Like, a backend may only support a few of functions in it, but still we want to know overall what is supported. Backend can have their own list and number if they need, or just use this two (like in dummy backend), meaning it support all fucntions.